src/Raven.Server/Documents/Handlers/Admin/Processors/Configuration/AbstractAdminConfigurationHandlerProcessorForPutSettings.cs

```diff
@@ -42,6 +43,9 @@ public override async ValueTask ExecuteAsync()
             long index = (await RequestHandler.Server.ServerStore.SendToLeaderAsync(command)).Index;
 
             await WaitForIndexNotificationAsync(index);
+            
+            if (LoggingSource.AuditLog.IsInfoEnabled)
+                RequestHandler.LogAuditFor(RequestHandler.DatabaseName, "PUT", "Database configuration changed");
```

Comment:
Same comment as in 5.4 - let's change it to `LogAuditFor(RequestHandler.DatabaseName, "CHANGE", "Database configuration");`

Reply:
Changed